source/plugins/utils/oms_common.rb

```diff
@@ -0,0 +1,1020 @@
+module OMS

```

Comment:
why do we need this ? We can just use a file logger and move on.

Reply:
We use the  common functions from OMS multiple places like hostname and logger etc.  since this code being already used & validated from the beginning so using the same functions.

Reply:
can we just keep those functions and remove others ?

Reply:
done.